yt/frontends/cf_radial/tests/test_cf_radial_pytest.py

```diff
@@ -0,0 +1,50 @@
+from pathlib import Path
+
+import numpy as np
+
+import yt
+from yt.frontends.cf_radial.data_structures import CFRadialDataset
+from yt.testing import requires_module
+
+
+def create_cf_radial_mock_gridded_ds(savedir: Path) -> Path:
+    # a minimal xarray dataset that should be identified as valid
+    import xarray as xr
+
+    file_to_save = savedir / "mock_gridded_cfradial.nc"
+
+    shp = (16, 16, 16)
+    xyz = {dim: np.linspace(0.0, 1.0, shp[idim]) for idim, dim in enumerate("xyz")}
+    da = xr.DataArray(data=np.ones(shp), coords=xyz, name="reflectivity")
+    ds_xr = da.to_dataset()
+    ds_xr.attrs["conventions"] = "CF/Radial"
+    ds_xr.x.attrs["units"] = "m"
+    ds_xr.y.attrs["units"] = "m"
+    ds_xr.z.attrs["units"] = "m"
+    ds_xr.reflectivity.attrs["units"] = ""
+
+    t = np.array([0.0, 1.0])
+    ds_xr = ds_xr.assign_coords({"time": t})
+    ds_xr["origin_latitude"] = xr.DataArray(np.zeros(t.shape), dims=("time"))
+    ds_xr["origin_longitude"] = xr.DataArray(np.zeros(t.shape), dims=("time"))
```

Comment:
```suggestion
    ds_xr["origin_latitude"] = xr.DataArray(np.zeros_like(t), dims=("time"))
    ds_xr["origin_longitude"] = xr.DataArray(np.zeros_like(t), dims=("time"))
```
also, I don't know what is the intention in `dims=("time")` but if you meant to write a tuple, note that you need `dims=("time",)`, otherwise you just have a parenthesised string, equivalent to `dims="time"` (because the syntactic element that makes a tuple isn't the parentheses, it's the comma)

Reply:
good catch, forgot the comma. i did mean a tuple... but looks like xarray doesn't require a tuple if it's a single element. i'll correct it.